package.json

```diff
@@ -8,20 +8,20 @@
   },
   "scripts": {
     "build": "npm run build:dist && npm run build:dist6",
-    "build:dist": "tsc --target es2017 --outDir dist",
-    "build:dist6": "tsc --target es2015 --outDir dist6",
-    "build:watch": "tsc --watch",
-    "clean": "rm -rf dist dist6",
+    "build:dist": "lb-tsc --target es2017 --outDir dist",
+    "build:dist6": "lb-tsc --target es2015 --outDir dist6",
+    "build:watch": "lb-tsc --watch",
```

Comment:
I am pretty unhappy with this change, all we did is change `tsc` to `lb-tsc`. The script is not simpler in any way that before. On the contrary, it adds one more level of abstraction that any developer using this package has to understand.

Before this change, anybody familiar with `tsc` could immediately understand what `npm run build:dist` is doing. With this change in place, they have to learn tricky details of how `lb-tsc` works.

For me, this is a net loss 👎

Reply:
This has been cleaned up by https://github.com/strongloop/loopback4-extension-starter/pull/17. PTAL.